tests/src/Traits/TripalCultivateImporterTestTrait.php

```diff
@@ -24,7 +24,9 @@ trait TripalCultivateImporterTestTrait {
    *         file in the temporary or public files directory.
    *     - content[string]: the content to copy into the file as a string
    *     - content[file]: an existing file in the fixtures directory to copy
-   *         the contents from
+   *         the contents from.
+   *     - content[fixturepath]: the path to Fixtures relative to the module the
+   *         method was called. Default to Fixtures directory of this module.
```

Comment:
This should not be a relative path but rather an absolute path.
```suggestion
   *     - content[fixturepath]: the absolute path to Fixtures directory
   *         containing the test file. This should end with a '/'.
   *         Default to Fixtures directory of this module.
```